examples/README.md

```diff
@@ -0,0 +1,8 @@
+# Example CloudEvents Data in JSON format with HTTP Headers
```

Comment:
While having cURL scripts is useful, I think it would also be useful to have JSON representations of the data part as plain JSON files too. That would be a lot easier to consume in tests etc. Over time we should probably work out a way of having multiple formats and then a system to generate all those formats from a single request. Obviously that can wait though.

Reply:
Added plain JSON.